src/coffee/mapping.coffee

```diff
@@ -367,6 +367,8 @@ class Mapping
     if _.has langH2i, attribName
       _.each langH2i[attribName], (index, language) ->
         val = row[index]
+        if val == ""
```

Comment:
I am translating null/undefined to an empty string. Also, the problem was that an empty string was not removed from searchKeywords (empty string is not allowed there)